plugins/events/stopBuilds.js

```diff
@@ -46,7 +46,13 @@ module.exports = () => ({
             }
 
             // Check permissions
-            const permissions = await user.getPermissions(pipeline.scmUri);
+            const permissions = await user.getPermissions(pipeline.scmUri).catch(err => {
+                if (err.statusCode === 404) {
+                    throw boom.notFound(`Repository ${pipeline.scmUri} does not exist`);
```

Comment:
Using `throw boom.boomify(err, { statusCode: err.statusCode });` style might be better for throwing error other than 404 error.

Reply:
I'll omit comments, but the same is true for other fixes.

Reply:
https://github.com/screwdriver-cd/screwdriver/blob/6ac28557524d0bbac61880e31c2402c91f3d7d98/plugins/pipelines/sync.js#L51

If we use `boom.boomify` style at the case above, error message will be `not found`. boomify seems to be able to override the message, but I think `boom.notFound` is easier.

In other cases, I fixed to use `boom.boomify` style.

Reply:
I fixed to use `boom.boomify`.